incidents/extract.py

```diff
@@ -0,0 +1,79 @@
+"""An extract script to retrieve live data from the
+Knowledgebase (KB) Real Time Incidents API"""
+
+from time import sleep
+from os import environ as ENV
+import logging
+from dotenv import load_dotenv
+import stomp
+
+
+
+def connect_and_subscribe(connection, admin:str, passcode:str, sub_topic:str):
```

Comment:
Great job with type hinting! Did you forget `connection`?

Reply:
Good catch! I will add it in.